site/content/en/menu/index.md

```diff
@@ -7,12 +7,12 @@ headless: true
   - **Quick Start**
     - [All in One Image]({{< relref "/docs/user/all-in-one-image" >}})
     - [Install]({{< relref "/docs/user/install" >}})
-    - [Kwok in Cluster]({{< relref "/docs/user/kwok-in-cluster" >}})
-    - [Kwok in Local]({{< relref "/docs/user/kwok-in-local" >}})
-    - [Kwokctl Manage Clusters]({{< relref "/docs/user/kwokctl-manage-cluster" >}})
+    - [kwok in Cluster]({{< relref "/docs/user/kwok-in-cluster" >}})
```

Comment:
Add below in `site/assets/_custom.scss` and also wrapping it in '`'
``` scss
code {
    padding: 0 .25rem;
    background: var(--gray-200);
    border-radius: .25rem;
    font-size: .875em;
}
```

Reply:
This snippet is added, not sure how to wrap it.